### DISCUSSION TITLE
Huggingface model

### DISCUSSION THREAD

**Discussion by 4rombusiness** [immich-app/immich#12030] [Q&A]
### The bug

Sorry if it sounds naive but when I do:

from transformers import AutoModel
model = AutoModel.from_pretrained("immich-app/ViT-H-14-378-quickgelu__dfn5b")

I get:
`File "/home/runner/DetailedAggressiveSystemintegrator/.pythonlibs/lib/python3.11/site-packages/transformers/models/auto/configuration_auto.py", line 1008, in from_pretrained
    raise ValueError(
ValueError: Unrecognized model in immich-app/ViT-H-14-378-quickgelu__dfn5b. Should have a model_type key in its config.json, or contain one of the following strings in its name: albert, align, altclip, ...`

### The OS that Immich Server is running on

NixOS

### Version of Immich Server

not related

### Version of Immich Mobile App

not related

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
not related
```

### Your .env content

```Shell
not related
```

### Reproduction steps

```bash
1. explain in the main issue box.
2.
3.
...
```

**Comment by bo0tzz** [comment]
What are you trying to do? I don't believe our models are intended for general use.

**Reply by 4rombusiness** [reply]
Saw it operating pretty smooth without much impact on resources. I just wanted to know more about it...
